packages/webapp/pages/onboarding.tsx

```diff
@@ -300,11 +345,15 @@ export function OnboardPage(): ReactElement {
     return onClickNext();
   };
 
-  const onSuccessfulRegistration = () => {
-    setActiveScreen(OnboardingStep.EditTag);
-  };
-
   const authOptionProps: AuthOptionsProps = useMemo(() => {
+    const onSuccessfulRegistration = () => {
+      if (interactiveFeedExp) {
+        setActiveScreen(OnboardingStep.InteractiveFeed);
+      } else {
+        setActiveScreen(OnboardingStep.EditTag);
+      }
```

Comment:
We should enroll the user from this point on only. So that not every visiting user would be enrolled. Only the ones that finishes the registration.